velox/exec/tests/UnnestTest.cpp

```diff
@@ -42,65 +126,15 @@ TEST_F(UnnestTest, basicArray) {
 }
 
 TEST_F(UnnestTest, arrayWithOrdinality) {
-  auto array = vectorMaker_.arrayVectorNullable<int32_t>(
-      {{{1, 2, std::nullopt, 4}},
-       std::nullopt,
-       {{5, 6}},
-       {{}},
-       {{{{std::nullopt}}}},
-       {{7, 8, 9}}});
-  auto vector = makeRowVector(
-      {makeNullableFlatVector<double>({1.1, 2.2, 3.3, 4.4, 5.5, std::nullopt}),
-       array});
-
-  auto op = PlanBuilder()
-                .values({vector})
-                .unnest({"c0"}, {"c1"}, "ordinal")
-                .planNode();
-
-  auto expected = makeRowVector(
-      {makeNullableFlatVector<double>(
-           {1.1,
-            1.1,
-            1.1,
-            1.1,
-            3.3,
-            3.3,
-            5.5,
-            std::nullopt,
-            std::nullopt,
-            std::nullopt}),
-       makeNullableFlatVector<int32_t>(
-           {1, 2, std::nullopt, 4, 5, 6, std::nullopt, 7, 8, 9}),
-       makeNullableFlatVector<int64_t>({1, 2, 3, 4, 1, 2, 1, 1, 2, 3})});
-  assertQuery(op, expected);
-
-  // Test with array wrapped in dictionary.
-  auto reversedIndices = makeIndicesInReverse(6);
-  auto vectorInDict = makeRowVector(
-      {makeNullableFlatVector<double>({1.1, 2.2, 3.3, 4.4, 5.5, std::nullopt}),
-       wrapInDictionary(reversedIndices, 6, array)});
-  op = PlanBuilder()
-           .values({vectorInDict})
-           .unnest({"c0"}, {"c1"}, "ordinal")
-           .planNode();
-
-  auto expectedInDict = makeRowVector(
-      {makeNullableFlatVector<double>(
-           {1.1,
-            1.1,
-            1.1,
-            2.2,
-            4.4,
-            4.4,
-            std::nullopt,
-            std::nullopt,
-            std::nullopt,
-            std::nullopt}),
-       makeNullableFlatVector<int32_t>(
-           {7, 8, 9, std::nullopt, 5, 6, 1, 2, std::nullopt, 4}),
-       makeNullableFlatVector<int64_t>({1, 2, 3, 1, 1, 2, 1, 2, 3, 4})});
-  assertQuery(op, expectedInDict);
+  std::vector<std::optional<std::vector<std::optional<int>>>> array = {
+      {{1, 2, std::nullopt, 4}},
+      std::nullopt,
+      {{5, 6}},
+      {{}},
+      {{{{std::nullopt}}}},
+      {{7, 8, 9}}};
+  //   testArrayWithOrdinality(false, array);
```

Comment:
Do we need to keep this test for outer=false?

Reply:
oh, I must comment it by mistake, will fix it, sorry for that